osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -111,7 +111,7 @@ private void dropdownSelectionChanged(ValueChangedEvent<Live<SkinInfo>> skin)
             configBindable.Value = skin.NewValue.ID.ToString();
         }
 
-        private void skinsChanged(IRealmCollection<SkinInfo> sender, ChangeSet changes, Exception error)
+        private void skinsChanged(IRealmCollection<SkinInfo> sender, ChangeSet changes, Exception error) => Schedule(() =>
```

Comment:
Not sure about scheduling the entire operation. Scheduling the children mutation and the `updateSelectedSkinFromConfig()` call makes sense, but is `sender` going to be valid for reading if it's delayed a frame?

Also, out of curiosity, how'd you find this? Is there a repro scenario that can be used?

Reply:
Fair call, I'll give that a try.

This repro'd randomly while I was running tests, no repro.